### DISCUSSION TITLE
第1节课笔记(1班)

### DISCUSSION THREAD

**Discussion by vansin** [InternLM/Tutorial#13] [笔记]
请在下方提交笔记链接

**Comment by xujinzh** [comment]
https://xujinzh.github.io/2024/01/03/ai-internlm-intro/

**Comment by UncleLLD** [comment]
https://blog.csdn.net/uncle_ll/article/details/135374647

**Comment by Xpg74138** [comment]
https://blog.csdn.net/qq_41776136/article/details/135375060

**Comment by gpt-dance** [comment]
https://zhuanlan.zhihu.com/p/676017235

**Comment by Gu-Xiao** [comment]
https://www.bilibili.com/read/cv29020774/?spm_id_from=333.999.0.0&jump_opus=1

**Comment by wudemoai** [comment]
https://blog.csdn.net/weixin_48478550/article/details/135383607?spm=1001.2014.3001.5501

**Comment by woxinyonghen** [comment]
https://blog.csdn.net/xingchengyonghen/article/details/135381127

**Comment by 10086zgyd** [comment]
https://blog.csdn.net/weixin_55546799/article/details/135389106?spm=1001.2014.3001.5501

**Comment by Andy1314Chen** [comment]
https://zhuanlan.zhihu.com/p/676098926

**Comment by blurmemo** [comment]
https://github.com/blurmemo/LLM/blob/main/note/InternLM/chap0.md

**Comment by ustbzgn** [comment]
https://blog.csdn.net/qq_46052444/article/details/135397254

**Comment by biscuit279** [comment]
https://blog.csdn.net/qq_46212981/article/details/135405135

**Comment by woml** [comment]
https://woml.github.io/Lesson_1

**Comment by random-zhou** [comment]
http://t.csdnimg.cn/amIs5

**Comment by strong-leaf** [comment]
https://www.zhihu.com/question/626788186/answer/3351147374

**Comment by gongxiucheng** [comment]
https://ckfajwu6zv.feishu.cn/docx/GuhFdjcnCoMJaZxCePbcA3zdnLg?from=from_copylink

**Comment by Marxlp** [comment]
https://github.com/Marxlp/internLM-course-notes/blob/main/lecture0.md

**Comment by formorebeat** [comment]
https://shimo.im/docs/loqeM11dwgFe50qn

**Comment by zpc-666** [comment]
https://blog.csdn.net/zpc1219/article/details/135416066

**Comment by RongjianLiang** [comment]
https://www.bilibili.com/read/cv29068412/?jump_opus=1

**Comment by grey2818** [comment]
https://zhuanlan.zhihu.com/p/676302227

**Comment by zdh19821025** [comment]
https://blog.csdn.net/zdh19821025/article/details/135417653

**Comment by raulming** [comment]
https://quilled-apparatus-d9a.notion.site/bea42c8702444e39a760cc0dee165fc9

**Comment by lds-coder** [comment]
https://blog.csdn.net/m0_73199148/article/details/135419958?app_version=6.2.6&csdn_share_tail=%7B%22type%22%3A%22blog%22%2C%22rType%22%3A%22article%22%2C%22rId%22%3A%22135419958%22%2C%22source%22%3A%22m0_73199148%22%7D&utm_source=app

**Comment by AXYZdong** [comment]
https://nuly9zxzf1.feishu.cn/docx/B4rFdjWMuoiQXwxqqVuc5waynlb?from=from_copylink

**Comment by LiyanJin** [comment]
https://blog.csdn.net/weixin_53443275/article/details/135468011

**Comment by xiaozhuang223** [comment]
https://zhuanlan.zhihu.com/p/676940900
